src/GrpcProtos/capture.proto

```diff
@@ -677,6 +677,18 @@ message ErrorEnablingUserSpaceInstrumentationEvent {
   bytes message = 2;
 }
 
+message FunctionThatFailedToBeInstrumented {
+  uint64 function_id = 1;
+  // This is a string, we use bytes to avoid UTF-8 validation.
+  bytes error_message = 2;
```

Comment:
Do we need still need the string here?  I feel that this is something the client should produce. Since there are two different errors that can be reported, consider adding a type.